src/platform/plugins/shared/dashboard/public/services/dashboard_backup_service.ts

```diff
@@ -31,10 +30,14 @@ const getPanelsGetError = (message: string) =>
     values: { message },
   });
 
```

Comment:
Nice idea to restructure this class more broadly. Maybe we'll take an even broader approach soon and remove the `service` entirely soon.